src/Microsoft.AspNet.Hosting/Program.cs

```diff
@@ -54,13 +54,20 @@ public void Main(string[] args)
                 shutdownHandle.Set();
             });
 
-            var ignored = Task.Run(() =>
+            var appLifeline = host.ApplicationServices.GetRequiredService<IApplicationLifetime>();
+            if (appLifeline.ApplicationStarted == null)
```

Comment:
Before, we always would print out "Started", then wait, then shut down. Is it deliberate that if there's a custom AppStarted handler that we don't to anything at all?